test/scss/_feature-targeting-test.scss

```diff
@@ -159,6 +160,15 @@
     @include mdc-elevation-overlay-fill-color(red, $query: $query);
     @include mdc-elevation-overlay-opacity(99%, $query: $query);
 
+    // Floating label
+    @include mdc-floating-label-core-styles($query: $query);
```

Comment:
Note that the `test:feature-targeting` check doesn't seem to be running on the CI because there are failures in the files that's in master. I've added my changes here for completeness.